utilities/data/getStoryList.ts

```diff
@@ -38,7 +38,7 @@ async function getStoryList({ path }: getStoryDataProps) {
 
   // For more related documentation see app/(storyblok)/[[...slug]]/page.tsx
   const sbParams: ISbStoriesParams = {
-    version: activeEnv === 'development' ? 'draft' : 'published',
+    version: 'published',
```

Comment:
For getStoryList, I think it's ok to alway only show the published stories both in editor or in builds